examples/resque_scheduler/cookbooks/custom-redis/attributes/default.rb

```diff
@@ -0,0 +1 @@
+default['redis']['utility_name'] = 'resque'
```

Comment:
For a *clustered* env,  this line should be replaced with at least:

```
default['redis'].tap do |redis|

  # Collect the redis instances in this array
  redis_instances = []

  # Run Redis on a named util instance
  # This is the default
  redis['utility_name'] = 'resque'
  redis_instances << redis['utility_name']
  redis['is_redis_instance'] = (
    node['dna']['instance_role'] == 'util' &&
    redis_instances.include?(node['dna']['name'])
  )
end
```

and for a *solo* env with:

```
default['redis'].tap do |redis|

  # Collect the redis instances in this array
  redis_instances = []

 # Run redis on a solo instance
  # Not recommended for production environments
  #redis['is_redis_instance'] = (node['dna']['instance_role'] == 'solo') 
end
```